data/fin.lean

```diff
@@ -88,3 +91,23 @@ by cases i; refl
   {n} {C : fin (succ n) → Sort*} {H0 Hs} (i : fin n) :
   @fin.cases n C H0 Hs i.succ = Hs i :=
 by cases i; refl
+
+def nat_expand_type {n} (i : fin n) (k) : fin (n + k) :=
```

Comment:
Would it be better to call `nat_expand_type` `raise_nat` or similar, since I believe it's conceptually similar to `raise` above?

Reply:
Done.